README.md

```diff
@@ -1,3 +1,34 @@
-# Uploadcare Binding for PubNub Chat Engine
+# Uploadcare Plugin for Chat Engine
 
-Easily add image uploads to chatrooms using UploadCare online service.
+Adds the ability to upload images in ChatEngine.Chats via UploadCare service
```

Comment:
Add a link to Uploadcare?

Reply:
I also think this is a chance to be casual and say "ChatEngine Chats" versus the literal ```ChatEngine.Chats```.